osmosis-1/osmosis.zone.json

```diff
@@ -1512,6 +1512,16 @@
       "osmosis_main": false,
       "osmosis_frontier": false,
       "osmosis_info": false
+    },
+    {
+      "chain_name": "juno",
+      "base_denom": "cw20:juno1qmlchtmjpvu0cr7u0tad2pq8838h6farrrjzp39eqa9xswg7teussrswlq",
+      "pools": {
+        "OSMO": 924
+      },
+      "osmosis_main": false,
+      "osmosis_frontier": true,
+      "osmosis_info": false
```

Comment:
Due to there being sufficient liquidity, we can make 'osmosis_info' tag true as well.